routes/html/index.js

```diff
@@ -1,20 +1,27 @@
 const Users = require("../../models/Users");
 const Games = require("../../models/Games");
 const sequelize = require("../../config/connection");
+const { withAuth } = require("../../utils/auth");
 
 const router = require("express").Router();
 
-router.get("/", (req, res) => {
+router.get("/", withAuth, async (req, res) => {
   const loggedIn = req.user;
```

Comment:
this is all wrong.

`req.user` isn't logged in. Its a User object. why are you calling it logged in?

Reply:
fixed in other PR